init-tools.cmd

```diff
@@ -10,8 +10,8 @@ if [%BUILDTOOLS_SOURCE%]==[] set BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/do
 set /P BUILDTOOLS_VERSION=< "%~dp0BuildToolsVersion.txt"
 set BUILD_TOOLS_PATH=%PACKAGES_DIR%Microsoft.DotNet.BuildTools\%BUILDTOOLS_VERSION%\lib\
 set PROJECT_JSON_PATH=%TOOLRUNTIME_DIR%\%BUILDTOOLS_VERSION%
-set PROJECT_JSON_FILE=%PROJECT_JSON_PATH%\project.json
-set PROJECT_JSON_CONTENTS={ "dependencies": { "Microsoft.DotNet.BuildTools": "%BUILDTOOLS_VERSION%" }, "frameworks": { "netcoreapp1.0": { } } }
+set PROJECT_JSON_FILE=%PROJECT_JSON_PATH%\project.csproj
+set PROJECT_JSON_CONTENTS=^^^<Project Sdk=^^^"Microsoft.NET.Sdk^^^"^^^>^^^<PropertyGroup^^^>^^^<TargetFramework^^^>netcoreapp1.0^^^</TargetFramework^^^>^^^<DisableImplicitFrameworkReferences^^^>true^^^</DisableImplicitFrameworkReferences^^^>^^^</PropertyGroup^^^>^^^<ItemGroup^^^>^^^<PackageReference Include=^^^"Microsoft.DotNet.BuildTools^^^" Version=^^^"%BUILDTOOLS_VERSION%^^^" /^^^>^^^</ItemGroup^^^>^^^</Project^^^>
```

Comment:
I think I would just call it something like init-tools.proj (or perhaps init-tools.msbuild to avoid clashing with other projs) and put it right next to the init-tools scripts.